app/src/main/res/values-bg/strings.xml

```diff
@@ -291,7 +291,7 @@
   <!--Possible folder states-->
   <string name="state_idle">Бездействие</string>
   <string name="state_scanning">Сканиране</string>
-  <string name="state_cleaning">Почистване</string>
+ 
```

Comment:
Blank new lines everywhere.

Reply:
They'll get deleted (overwritten) as tx pull will take place fetching up new translations from transifex. Btw can you please push out new strings to transifex with tx push after merge, please?

Reply:
I don't want to have a PR that adds a bunch of meaningless lines in the history, so let's remove these in this PR. I'll try to update the translations post merge.

Reply:
Ok, makes sense keeping the history clean.  Build may fail because of the "beetling" translation strings, but we are fine.